server/kadalu_quotad/quotad.py

```diff
@@ -130,19 +203,33 @@ def start():
     Start Quota Manager
     """
     first_time = True
+    automatic_bricks = []
+    automatic_pass = 0
     while True:
+        try_automatic = True
         brick_path = os.environ.get("BRICK_PATH", None)
         if brick_path is not None:
+            try_automatic = False
             crawl(brick_path)
         try:
             with open(CONFIG_FILE) as conf_file:
+                try_automatic = False
                 config_data = json.loads(conf_file.read().strip())
                 for brick in config_data.get('bricks', None):
                     crawl(brick)
         except:    # noqa # pylint: disable=bare-except
             # Ignore all errors
             pass
 
+        if try_automatic:
+            # Getting the bricks is expensive, do it roughly
```

Comment:
Volume info command is a local operation that reads from a local file in `/var/lib/glusterd`. The short interval should be fine.

Reply:
This process takes 3 seconds on a 'server class' machine which isn't particularly loaded but has 16 volumes with 5 bricks locally. I felt this was too much to do on every loop, so I've left it in.